### PR TITLE
Feature/hero component

### PR DESCRIPTION
**Pull request by pecfw**
## Description

Updated Hero component on homepage to be inline with the new style.

## Checks

- [x] Ticket meets all Acceptance Criteria

- [x] Responsive on Desktop, Mobile and Tablet

- [x] 90% test coverage

- [ ] Documentation updated

## Screenshots
Screenshot 2020-04-02 at 14 36 42
Screenshot 2020-04-02 at 14 37 09
Screenshot 2020-04-02 at 14 38 25

### CODE REVIEW

File: src/components/HeroHomepage/HeroHomepage.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  title,
+  subtitle,
+}) => {
+  const classes = useStyles()
+  return (
+    <Box component="section" className={classes.heroContainer}>
+      <Box className={classes.imageWrapper}>
+        <Image fixed={fixed} alt={heroImageTitle} />
+      </Box>
+      <Box maxWidth="1620" className={classes.contextWrapper}>
```
Comment (Akallabet) [inline src/components/HeroHomepage/HeroHomepage.js:84]: Could we remove `maxWidth`?

File: src/components/HeroHomepage/HeroHomepage.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+const HeroHomepage = ({
+  image: {title: heroImageTitle, fixed},
+  title,
+  subtitle,
+}) => {
+  const classes = useStyles()
+  return (
+    <Box component="section" className={classes.heroContainer}>
+      <Box className={classes.imageWrapper}>
+        <Image fixed={fixed} alt={heroImageTitle} />
```
Comment (Akallabet) [inline src/components/HeroHomepage/HeroHomepage.js:82]: The image should be fluid

File: src/pages/index.js [hunk 3]
```diff
@@ -100,8 +100,8 @@ export const query = graphql`
         subtitle
         image {
           title
-          fixed(height: 605) {
-            ...GatsbyContentfulFixed_withWebp
+          fluid {
```
Comment (andgreendigital) [inline src/pages/index.js:103]: Specify a max width please! 🙂

File: src/pages/index.js [hunk 4]
```diff
@@ -99,13 +97,11 @@ export const query = graphql`
...
-        text {
-          json
-        }
+        subtitle
         image {
           title
-          fixed(height: 605) {
-            ...GatsbyContentfulFixed_withWebp
+          fluid {
+            ...GatsbyContentfulFluid
```
Comment (andgreendigital) [inline src/pages/index.js:104]: Why was the webp version removed?

Comment (pecfw) [reply src/pages/index.js:104]: This is using our standard that is being used for section card

**Pull request merged**